app/src/main/res/layout-land/submitted_answer_item.xml

```diff
@@ -58,7 +58,7 @@
       android:textSize="16sp"
       android:visibility="gone"
       app:layout_constrainedWidth="true"
-      app:layout_constraintEnd_toEndOf="@{viewModel.isCorrectAnswer ? @id/answer_tick_guideline : ConstraintSet.PARENT_ID}"
+      app:layout_constraintEnd_toEndOf="@{viewModel.isCorrectAnswer &amp;&amp; !viewModel.hasConversationView ? @id/answer_tick_guideline : ConstraintSet.PARENT_ID}"
```

Comment:
Change done which was missed in #3172 but this file will eventually get removed in coming PRs. Just fixed it so that there is no confusion while checking the screenshots.